### DISCUSSION TITLE
Looking for feedback

### DISCUSSION THREAD

**Discussion by Fevol** [Fevol/obsidian-translate#24] [General]
Hello everyone, I'm currently taking a break on this plugin to focus on other things, there might not be updates on this plugin for at least the coming two weeks.

Now for the question at hand: I'm very interested to hear some feedback on how you _use_ this plugin, as I'm not sure where I should be focusing my efforts in the future. Below are some types of feedback I'm most curious about, though you can write whatever you want!

- What do you use the plugin for?
- What features do you use a lot?
- What would make your workflow more efficient?
- Are there features that are hard to understand? Difficult to use?
- Have you been encountering performance issues while using the plugin? (Slower start-up times, lagging, ...)

**Comment by fgbyte** [comment]
the plugin start to do this error with the openAI keys
net::ERR_SSL_CLIENT_AUTH_CERT_NEEDED
